ops/reth-cross-test/verification/rollup_checker.py

```
@@ -0,0 +1,90 @@
+    """
+    l1_w3 = Web3(Web3.HTTPProvider(config.L1_RPC, request_kwargs={"timeout": 10}))
+
+    # Find CommitBatch events on L1
+    try:
+        logs = l1_w3.eth.get_logs({
+            "fromBlock": 0,
+            "toBlock": "latest",
+            "address": Web3.to_checksum_address(ROLLUP_ADDRESS),
+        })
+        batches_found = len(logs)
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)